components/eamxx/tests/uncoupled/ml_correction/ml_correction_standalone.cpp

```diff
@@ -71,6 +73,8 @@ TEST_CASE("ml_correction-stand-alone", "") {
           num_cols * num_levs, qv.data(), py::str{}),
       num_cols, num_levs);
   py::gil_scoped_release no_gil;
+  int fpe_mask = ekat::get_enabled_fpes();
```

Comment:
What's this fpe_mask for?

Reply:
It should get passed to `ekat::enable_fpes`, updated